Misc/NEWS.d/next/Build/2019-04-02-09-25-23.bpo-36503.0xzfkQ.rst

```diff
@@ -0,0 +1,2 @@
+Remove references to "aix3" and "aix4"
```

Comment:
```suggestion
Remove references to "aix3" and "aix4".
```

Reply:
I guess I could have just pressed the button above. Next time :) - Changed, in any case.